<?xml version="1.0"?>
<launch>

	<rosparam file="$(find cob_controller_configuration_gazebo)/ros/controller/tray_controller.yaml" command="load" />

	<node name="tray_controller_spawner" pkg="controller_manager" type="spawner" args="tray_controller" />

	
	<!-- start simulated phidget sensors -->
	<node ns="tray_sensors" name="tray_sensors" pkg="cob_phidgets" type="simulated_range_sensors.py" output="screen"/>

</launch>
